docs/man/docker.1.md

```diff
@@ -56,6 +56,9 @@ To see the man page for a command run **man docker <command>**.
 **--exec-opt**=[]
   Set exec driver options. See EXEC DRIVER OPTIONS.
 
+**--exec-root**=""
+  Path to use as the root of the Docker execdriver. Default is `/var/run/docker`.
+
```

Comment:
Is this option only available for the native driver, or also for LXC? That could be important information to add (also in the other suggested parts of the documentation)